lib/assets/javascripts/twine.coffee

```diff
@@ -265,6 +282,13 @@
   isKeypath = (value) ->
     value not in ['true', 'false', 'null', 'undefined'] && keypathRegex.test(value)
 
+  isDataAttribute = (value) ->
```

Comment:
Having a separate method for this might take away the performance benefit of doing it